mteb/evaluation/evaluators/ClassificationEvaluator.py

```diff
@@ -274,6 +274,7 @@ def __call__(
         X_train = model.encode(
             self.sentences_train,
             task_name=self.task_name,
+            task_sample_data=self.sentences_train,
```

Comment:
This still seems like a thing that is not needed and only customized to one specific model. I would prefer it if we didn't do it.